package_manifest.yaml

```diff
@@ -29,9 +29,31 @@ all:
         dist: '.el8'
         macros:
           foremandist: "{{ foremandist }}"
+    copr_project_user: "theforeman"
```

Comment:
there is no such user (but there is a `@theforeman` group), which makes me wonder if this variable is really more a `copr_project_namespace`?